x/scheduler/module.go

```diff
@@ -99,6 +99,14 @@ type AppModule struct {
 	bankKeeper    types.BankKeeper
 }
 
+func (am AppModule) IsOnePerModuleType() {
+	//TODO implement me
```

Comment:
When is this coming?

Reply:
There is no usage for this function but required by sdk keeper struct.

[Ref](https://github.com/cosmos/cosmos-sdk/blob/7d5c2dbae68530bc9070983ce0b2c8a1efe23412/x/bank/module.go#L100) usage in sdk bank module

Reply:
In that case, remove the TODO and replace with the contents of your comment, including the reference link. Thanks.

Reply:
Done